### ISSUE TITLE
New tag - Incomplete Penetrance

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#2836]
**Is your feature request related to a problem? Please describe.**
NA

**Describe the solution you'd like**
We would like to request for a new variant tag under functional data, 'Incomplete Penetrance'. This can be colored a light, transclucent pink. 

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by hanars** [comment]
Are you sure you want this to be functional data? Everything else tracked there tracks specific evidence that exists for that variant that points to why it is a discovery variant, so this seems a bit out of place. Also this tag would only be available on discovery variants and this seems like something you might want to tag during analysis before you were confident it was the discovery

Assuming this does need to be a functional data tag, I need a couple of clarifications:

1. Which section should this be in in the dropdown for adding functional tags? The current sections are Functional Data, Functional Scores, and Additional Kindreds (Literature, MME) but it could also be in its own section
2. Functional tags have additional metadata associated with them, for most it is just free text Notes but it could also be a specific score or value. Should this tag have any special value associated with it?

**Comment by lynnpais** [comment]
Yes, we would like this to be a functional tag. The idea is to use it in cases where one is confident of incomplete penetrance (IP), not just suspected IP. Having it in the functional section would mandate the use of a discovery tag such as Known Gene for Phenotype tag or another Tier 1/2 tag, requiring some level of confidence in the dx.

Regarding your specific questions,
1. It can go in a new section at the bottom, Additional Information with text: 
Variant has been shown to be disease-causing (in literature, functional studies, etc.) but one or more individuals in this family with the variant do not present with clinical features of the disorder.
2. A free text section can be included.

<img width="871" alt="Screen Shot 2022-07-15 at 12 05 45 PM" src="https://user-images.githubusercontent.com/37872186/179262839-15d8c931-b541-4cac-955f-371ba090fb46.png">

<img width="835" alt="Screen Shot 2022-07-15 at 12 05 54 PM" src="https://user-images.githubusercontent.com/37872186/179262860-f11136f4-7ea5-49a5-950e-4b330655688d.png">


Note: this is for internal data gathering purposes only, not for CMG reporting. Mentioning if this affects the workflow.

**Comment by hanars** [comment]
added
